src/istio/utils/attribute_names.cc

```diff
@@ -70,6 +70,7 @@ const char AttributeName::kConnectionEvent[] = "connection.event";
 // Context attributes
 const char AttributeName::kContextProtocol[] = "context.protocol";
 const char AttributeName::kContextTime[] = "context.time";
+const char AttributeName::kContextRoutingErrorReason[] = "context.routing_error_reason";
```

Comment:
1)  should we call it request, instead of context.  It is about the request.?
2) response_flags may include errors other than route, such as rate limiting, fault injection.  any errors from envoy proxy.  so it is better not call it "routing_error_reason".

Reply:
good point. `response.error_reason` ?